libs/ui-lib/lib/cim/components/Agent/BMCForm.tsx

```diff
@@ -183,7 +184,7 @@ const emptyValues: AddBmcValues = {
   password: '',
   bootMACAddress: '',
   disableCertificateVerification: true, // TODO(mlibra)
-  online: false, // TODO(mlibra)
+  online: false,
```

Comment:
Should it be 'true' by default?

Reply:
IIUC the bug is caused by the fact that the BMH is created with `online: false` which causes the host to never go through discovery because it does not get booted. That is why I think it should be `online: true` by default. Is there ever an use case in CIM when user would want the host to be in `online: false`?